lib/cypress_on_rails/vcr/use_cassette_middleware.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'base_middleware'
+
+module CypressOnRails
+  module Vcr
+    # Middleware to handle vcr with use_cassette
+    class UseCassetteMiddleware < BaseMiddleware
+      def initialize(app, vcr = nil)
+        @app = app
+        @vcr = vcr
+      end
+
+      def call(env)
+        return @app.call(env) if should_not_use_vcr?
```

Comment:
This is a workaround to skip behavior if VCR is already in use (rspec, for example)